src/components/menu/Menu.stories.js

```diff
@@ -0,0 +1,173 @@
+import UidsMenu from './Menu.vue';
+
+// More on default export: https://storybook.js.org/docs/vue/writing-stories/introduction#default-export
+export default {
+  title: 'Components/Menu',
+  parameters: {
+    docs: {
+      source: {
+        code: null,
+      },
+    },
+  },
+  component: UidsMenu,
+  tags: ['autodocs'],
+  // More on argTypes: https://storybook.js.org/docs/vue/api/argtypes
+  argTypes: {
+    variant: {
+      control: { type: 'select' },
+      options: ['horizontal', 'vertical'],
+    },
+    theme: {
+      control: { type: 'select' },
+      options: ['', 'light', 'on-dark'],
+    },
+  },
+};
+
+// More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
+const Template = (args) => ({
+  // Components used in your story `template` are defined in the `components` object
+  components: { UidsMenu },
+  // The story's `args` need to be mapped into the template through the `setup()` method
+  setup() {
+    return { args };
+  },
+  // And then the `args` are bound to your component with `v-bind="args"`
+  template: `
+    <uids-menu v-bind="args" />
+  `,
+});
+
+// Sample menu data with icons
+const horizontalMenuItems = [
+  { 
+    label: 'Home', 
+    url: '#home',
+    icon: 'fas fa-home',
+    active: true
+  },
+  { 
+    label: 'About', 
+    url: '#about',
+    icon: 'fas fa-info-circle'
+  },
+  { 
+    label: 'Services', 
+    url: '#services',
+    icon: 'fas fa-briefcase',
+    children: [
+      { label: 'Web Design', url: '#web-design' },
+      { label: 'Development', url: '#development' },
+      { label: 'Consulting', url: '#consulting' },
+    ]
+  },
+  { 
+    label: 'Contact', 
+    url: '#contact',
+    icon: 'fas fa-envelope'
+  },
+];
+
+const verticalMenuItems = [
```

Comment:
I don't think it's necessary to have two separate sample data sets for vertical and horizontal menus. We can use one for both.